Java/core/src/main/java/com/amazon/randomcutforest/tree/BoundingBox.java

```diff
@@ -79,8 +81,10 @@ protected BoundingBox(final double[] minValues, final double[] maxValues) {
         dimensions = minValues.length;
     }
 
-    public int getDimensions() {
-        return dimensions;
+    public BoundingBox copy() {
```

Comment:
I think I'd like this better as a copy constructor, rather than starting a new convention where we define a `copy()` method to create a copy.

Reply:
Fixed, changed to copyBox.